stable/moodle/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 name: moodle
-version: 1.0.3
+version: 1.0.4
```

Comment:
the version bump in the mariadb dependency warrants a major version bump in the moodle chart as it makes the release backwards incompatible.

Reply:
I don't understand why updating the dependency requires a major version bump in the chart. The chart continues working without any change.

Reply:
that changes are for mariadb 4.x, but it is true that with the 3.x version the moodle upgrade does not work well. At the moment I am undoing the mariadb updates changes and just expose the readiness/liveness probes

Reply:
We'll update MariaDB in a different way. Reverted f703b9f